doc/design/abs_backup.md

```diff
@@ -0,0 +1,73 @@
+# Backups using Azure Blob Service (ABS)
+
+In a very similar fashion to backups using [S3](./s3_backup.md), we want to have an option to store backups in ABS.
```

Comment:
Not sure if this description here is necessary; the title `Backups using Azure Blob Service (ABS)` already mentions what this doc is about.